packages/renderer/src/lib/preferences/PreferencesRenderingItemFormat.svelte

```diff
@@ -167,7 +167,7 @@ async function onChange(recordId: string, value: boolean | string | number): Pro
     {:else}
       <NumberItem
         record={record}
-        value={typeof recordValue === 'number' ? recordValue : getNormalizedDefaultNumberValue(record)}
+        value={typeof givenValue === 'number' ? givenValue : (typeof recordValue === 'number' ? recordValue : getNormalizedDefaultNumberValue(record))}
```

Comment:
This line is not clear / readable, I don't understand easily what it is doing, we should either have a dedicated function or a derived state (or something else?)